lib/ui/profile/Profile.dart

```diff
@@ -1,22 +1,91 @@
+import 'package:farmsmart_flutter/model/bloc/chatFlow/CreateAccountFlow.dart';
 import 'package:farmsmart_flutter/model/entities/ImageURLProvider.dart';
 import 'package:farmsmart_flutter/model/entities/loading_status.dart';
-import 'package:farmsmart_flutter/ui/common/ProfileAvatar.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheet.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheetListItem.dart';
 import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
 import 'package:farmsmart_flutter/ui/common/LoadableViewModel.dart';
+import 'package:farmsmart_flutter/ui/common/ProfileAvatar.dart';
 import 'package:farmsmart_flutter/ui/common/RefreshableViewModel.dart';
 import 'package:farmsmart_flutter/ui/common/ViewModelProviderBuilder.dart';
+import 'package:farmsmart_flutter/ui/common/image_picker.dart';
+import 'package:image_picker/image_picker.dart' as ImagePickerLib;
+import 'package:farmsmart_flutter/ui/common/modal_navigator.dart';
 import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/common/webview.dart';
+import 'package:farmsmart_flutter/ui/profile/FarmDetailsListItem.dart';
 import 'package:farmsmart_flutter/ui/profile/ProfileListItem.dart';
 import 'package:flutter/material.dart';
 import 'package:intl/intl.dart';
+import 'package:share/share.dart';
 
+import 'FarmDetails.dart';
 import 'SwitchProfileList.dart';
 
 class _LocalisedStrings {
-  static String activeCrops() => Intl.message('Active crops');
-  static String completedCrops() => Intl.message('Completed');
-  static String buttonTitle() => Intl.message('Switch Profile');
+  static activeCrops() => Intl.message('In progress crops');
+
+  static editFarmDetails() => Intl.message('Edit Farm Details');
+
+  static completedCrops() => Intl.message('Completed');
+
+  static buttonTitle() => Intl.message('Switch Profile');
+
+  static switchLanguage() => Intl.message('Switch Language');
+
+  static yourFarmDetails() => Intl.message('Your Farm Details');
+
+  static createNewProfile() => Intl.message('Create New Profile');
+
+  static inviteFriends() => Intl.message('Invite Friends');
+
+  static privacyPolicy() => Intl.message('Privacy Policy');
+
+  static termsOfUse() => Intl.message('Terms of use');
+
+  static logout() => Intl.message('Logout');
+
+  static removeProfile() => Intl.message('Remove Profile');
+
+  static confirm() => Intl.message('Confirm');
+
+  static cancel() => Intl.message('Cancel');
+
+  static pickImageFromGallery() => Intl.message('Pick image from gallery');
+
+  static takePictureFromCamera() => Intl.message('Take picture from camera');
+
+  //TODO Add the correct text & Google Play Link
+  static shareText() => Intl.message(
+      'Join to the new smart farming app - FarmSmart  \n https://linktogoogleplay.com');
+}
+
+class _Strings {
+  static final englishAction = "English";
+  static final swahiliAction = "Swahili";
```

Comment:
What happens if we have swahili as default language, it still showing language names in english right?

Reply:
The language strings should all be in the native language (so they don't need to run via the translation system) We need to use the translated string for "Swahili"google says its "Kiswahili"

Reply:
done